src/lib.rs

```diff
@@ -175,6 +175,9 @@ pub unsafe fn new_unchecked<'rx, 'tx>(
             // Duplex mode
             .dm()
             .set_bit()
+            // IPv4 checksum offload
+            .ipco()
```

Comment:
For IPv4 checksum offload support, Enhanced Descriptors must be enabled on `f4` and `f7` parts (`f1` parts don't seem to have this requirement, nor support for enhanced descriptors).